skimage/feature/_canny.py

```diff
@@ -11,11 +11,13 @@
 All rights reserved.
 Original author: Lee Kamentsky
 """
+from __future__ import division, print_function, absolute_import
 
 import numpy as np
 import scipy.ndimage as ndi
-from scipy.ndimage import (gaussian_filter,
-                           generate_binary_structure, binary_erosion, label)
+from scipy.ndimage import (generate_binary_structure, binary_erosion, label)
+
```

Comment:
I prefer to group imports by external packages, (newline), internal imports. Especially when you get more than 5 lines, it's easier to quickly scan with separation.